articles/multifactor-authentication/google-auth/admin-guide.md

```diff
@@ -0,0 +1,116 @@
+# Google Authenticator for Administrators
+
+::: panel-info Note:
+This page refers to using Google Authenticator instead of Auth0 Guardian. Google Authenticator can also be used with Guardian(your users choose which to use) when **Push Notifications** is enabled. [Click here for more information on Guardian.](/multifactor-authentication/guardian)
+:::
+
+## Enabling Google Authenticator for MFA
+
+To turn on Google Authenticator for two-step verification, first visit the [Multifactor Auth](${uiURL}/#/guardian) page from the dashboard. Then click on the link to use a different provider.
+
+![](/media/articles/mfa/change-provider.png)
+
+Then you can use the slider to turn on Google Authenticator.
+
+![](/media/articles/mfa/toggle-google-auth.png)
+
+## Google Authenticator Supported Devices
```

Comment:
I know that the provider is called Google Authenticator but, actually it is compatible with almost all code generator apps on the market. We might want to mention that.